risc0/bigint2/src/field/mod.rs

```diff
@@ -15,6 +15,8 @@
 #[cfg(test)]
 mod tests;
 
+pub mod syscall_field;
```

Comment:
Could call it `ffi` (Foreign-function interface)


```suggestion
pub mod ffi;
```

Reply:
ok, for now I changed it to `ffi_field`, I'm not sure what convention is considered best because later we could also have `ffi_ec` in the other directory of this bigint2 crate.

Reply:
If we wanted this for EC we could put it in the same file; there's no concern about name collisions as we have to prefix everything anyway. So I think ffi by itself would be fine

Reply:
Since this is already nested in a subdir, I think `ffi` is sufficient.